testsupport/init.go

```diff
@@ -23,12 +23,18 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+type Awaitilities struct {
```

Comment:
just curious: can't we have this struct in the same `wait` package as `HostAwaitility` and `MemberAwaitility`?

Reply:
That's a good point, it's probably a better place for it. Moved: https://github.com/codeready-toolchain/toolchain-e2e/pull/388/commits/fff83ef4317afc41269587072474e3509ee1ff9d